.env.example

```diff
@@ -7,3 +7,7 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+
+#NODE_ENV=production
```

Comment:
[Question] Is it needed?

Reply:
https://github.com/probot/probot/blob/master/docs/deployment.md